server/rosetta/cosmos/conversion/from_rosetta.go

```diff
@@ -0,0 +1,87 @@
+package conversion
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/gogo/protobuf/jsonpb"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+
+	"github.com/cosmos/cosmos-sdk/server/rosetta"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// RosettaOperationsToSdkMsg converts rosetta operations to sdk.Msg and coins
+func RosettaOperationsToSdkMsg(interfaceRegistry jsonpb.AnyResolver, ops []*types.Operation) ([]sdk.Msg, sdk.Coins, error) {
+	var feeAmnt []*types.Amount
+	var newOps []*types.Operation
+	if len(ops)%2 == 0 {
```

Comment:
do they come in pairs? which ones? this needs a comment as it's not intuitive to the reader

Reply:
this will be changed, it was before the extension for custom messages